student.rb

```diff
@@ -1,10 +1,22 @@
+require_relative 'person'
+require_relative 'teacher'
+require_relative 'classroom'
+require_relative 'book'
+
 class Student < Person
-  def initialize(name: 'Unknown', age: 0, parent_permission: true, classroom: 'Unknown')
-    super(name: name, age: age, parent_permission: parent_permission)
+  attr_accessor :classroom
+
+  def initialize(classroom)
+    super('kenny', 10)
     @classroom = classroom
   end
 
   def play_hooky
     '¯(ツ)/¯'
   end
+
+  def classrooms=(classroom)
```

Comment:
- [ ] You have made a typo here its classroom, not classrooms.

Reply:
Good evening @sja-thedude, it gives me an error since i had already defined classroom already on line 7

Reply:
Thats fine, I thought is typo